builder/build.sh

```diff
@@ -64,14 +64,17 @@ patch_r() {
   cd /tmp/R-${1}
 
   if [ -f "/patches/R-${1}.patch" ]; then
+      echo "Applying patch /patches/R-${1}.patch"
```

Comment:
Added because at first, I couldn't figure out why removing the fedora-40 patches broke the fedora-41 builds